aospy/test/aospy_test_objs/main.py

```diff
@@ -0,0 +1,213 @@
+#! /usr/bin/env python
```

Comment:
I don't think we should include all the parsing logic from main.  Can we just create the `Calc` objects and then run them?